.github/workflows/lint.yml

```diff
@@ -1,4 +1,4 @@
----   # Rust lint related checks
+--- # Rust lint related checks
```

Comment:
nit: why the change? It's getting yaml linter to complain

Reply:
Looks like `yamllint` is fine - it was just autoformatting in VSCode :)